tests/foreman/ui/test_org.py

```diff
@@ -287,35 +207,35 @@ def test_negative_create_3(self, org_name):
             self.assertIsNotNone(self.org.search(org_name))
             self.org.create(org_name)
             error = session.nav.wait_until_element(
-                common_locators["name_haserror"])
+                common_locators['name_haserror'])
             self.assertIsNotNone(error)
 
     # Positive Delete
 
     @data(*generate_strings_list())
-    def test_positive_delete_1(self, org_name):
+    def test_positive_delete(self, org_name):
         """@test: Create organization with valid values then delete it.
 
         @feature: Organizations Positive Delete test.
 
-        @assert: organization is deleted
+        @assert: Organization is deleted successfully
 
         """
         with Session(self.browser) as session:
             make_org(session, org_name=org_name)
             self.assertIsNotNone(self.org.search(org_name))
-            self.org.remove(org_name, really=True)
+            self.org.remove(org_name)
```

Comment:
I changed signature for original method and made that parameter optional with default value = 'True':
```
def remove(self, org_name, really=True):
```